core/src/main/scala/org/http4s/sbt/Http4sOrgPlugin.scala

```diff
@@ -41,39 +41,15 @@ object Http4sOrgPlugin extends AutoPlugin {
 
   val githubActionsSettings: Seq[Setting[_]] =
     Seq(
-      spiewakMainBranches := Seq("main"),
+      tlCiReleaseBranches := Seq("main"),
       githubWorkflowJavaVersions := List("8", "11", "17").map(JavaSpec.temurin(_)),
-      githubWorkflowEnv += ("JABBA_INDEX" -> "https://github.com/typelevel/jdk-index/raw/main/index.json"),
-      githubWorkflowBuild := Seq(
-        WorkflowStep
-          .Sbt(List("scalafmtCheckAll", "scalafmtSbtCheck"), name = Some("Check formatting")),
-        WorkflowStep.Sbt(List("headerCheckAll"), name = Some("Check headers")),
-        WorkflowStep.Sbt(List("test:compile"), name = Some("Compile")),
-        WorkflowStep.Sbt(List("mimaReportBinaryIssues"), name = Some("Check binary compatibility")),
+      githubWorkflowBuild +=
         WorkflowStep.Sbt(
-          List("unusedCompileDependenciesTest"),
-          name = Some("Check unused compile dependencies")),
```

Comment:
This is something not currently covered in sbt-typelevel. I'm a bit confused, are projects actually using this? If so I should add it back.

Reply:
http4s should be.  I think it's good POM hygiene, but it's one of my less popular opinions.

Reply:
I've never used it but seems like a good idea. I don't see it in the http4s CI rn:
https://github.com/http4s/http4s/blob/0671d49d246639e3edad89d7f8e1ba979d9a7658/.github/workflows/ci.yml#L100-L125

Reply:
We had the explicit dependencies check in addition to the unused dependencies check.  The former was very controversial, but I'd like to reinstate the latter.